packages/now-static-build/src/frameworks.ts

```diff
@@ -317,4 +318,5 @@ export interface Framework {
   defaultRoutes?: Route[] | ((dirPrefix: string) => Promise<Route[]>);
   minNodeRange?: string;
   beforeBuildHook?: (entrypointDir: string) => Promise<void>;
+  cachePaths?: string[];
```

Comment:
From my other comment, let's change this
```suggestion
  cachePattern?: string;
```